### PR TITLE
Bump to 0.12.30 before publishing.

### PR DESCRIPTION
**Pull request by munificent**
Publishing the current stuff so I can roll it internally.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-## 0.12.29+1
+## 0.12.30
```
Comment (nex3) [inline CHANGELOG.md]: Why not 0.12.29+1?

Comment (munificent) [reply CHANGELOG.md]: The pubspec was already at 0.12.30-dev.

Comment (nex3) [reply CHANGELOG.md]: That's just because `0.12.29+1-dev` isn't valid. There aren't any changes in the changelog that would require a minor version bump.

Comment (munificent) [reply CHANGELOG.md]: Ah, I see. Done.

**Pull request merged**